src/modules/savedjobs/repository/savedjobs.repository.ts

```
@@ -0,0 +1,42 @@
+      .skip((pageOptionsDto.page - 1) * pageOptionsDto.take)
+      .take(pageOptionsDto.take);
+
+    if (filters.userId) {
+      queryBuilder.andWhere('user.id = :userId', { userId: filters.userId });
+    }
+
+    if (filters.jobId) {
+      queryBuilder.andWhere('savedJob.jobId = :jobId', { jobId: filters.jobId });
+    }
+
+    const itemCount = await queryBuilder.getCount().catch(handleError);
```

ESLint

[error] Delete `␍`

(prettier/prettier)